cpp/proto/serializer.h

```diff
@@ -26,71 +26,25 @@ cert_trans::serialization::SerializeResult CheckSctExtensionsFormat(
 void WriteSctExtension(const repeated_sct_extension& extension,
                        std::string* output);
 
-class TLSDeserializer {
- public:
-  // We do not make a copy, so input must remain valid.
-  // TODO(pphaneuf): And so we should take a string *, not a string &
-  // (which could be to a temporary, and not valid once the
-  // constructor returns).
-  explicit TLSDeserializer(const std::string& input);
-
-  bool ReachedEnd() const {
-    return bytes_remaining_ == 0;
-  }
+namespace serialization {
 
-  cert_trans::serialization::DeserializeResult ReadSCT(
-      ct::SignedCertificateTimestamp* sct);
+namespace internal {
```

Comment:
You can skip whitespace between consecutive `namespace` openings/closings (really, it'd be nicer if we could just go `namespace serialization::internal {`, but here we are).